src/theme/index.ts

```diff
@@ -36,6 +36,13 @@ const colors = {
   },
 }
 
+const layerStyles = {
```

Comment:
why use `layerStyle` instead of `textStyle`?  Tbh I am not sure of the difference - chakra docs indicate that layerStyle should be used for things like `color`, `border`, `boxShadow`, where textStyle should be used for `fontFamily`, `lineHeight`, `fontSize`, etc. 

https://chakra-ui.com/docs/styled-system/features/text-and-layer-styles


Do you think it is necessary to separate them out?

Reply:
Please see [2f991de](https://github.com/threshold-network/splash-page/pull/22/commits/2f991de3e19d390853ac27ca127a62a04e0c292b).